src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Windows.cs

```diff
@@ -796,7 +796,7 @@ public static unsafe SocketError Poll(SafeSocketHandle handle, int microseconds,
         public static unsafe SocketError Select(IList checkRead, IList checkWrite, IList checkError, int microseconds)
         {
             const int StackThreshold = 64; // arbitrary limit to avoid too much space on stack
```

Comment:
Apparently the compiler we're using in corefx doesn't contain the fix for https://github.com/dotnet/roslyn/issues/36091.  Oh well.

Reply:
Because it was merged just 18 days ago?

Reply:
Yes.

Reply:
should we update it? https://github.com/dotnet/arcade/blame/master/eng/Versions.props#L33

Reply:
> should we update it?

In general I think we should stay up-to-date with compiler bits.  However, I've not followed what releases have been published, are recommended, etc.